package.json

```diff
@@ -172,7 +171,8 @@
     "stylelint-order": "^0.6.0",
     "webpack": "^3.3.0",
     "webpack-bundle-analyzer": "^2.8.3",
-    "webpack-dev-server": "^2.5.1"
+    "webpack-dev-server": "^2.5.1",
+    "worker-loader": "^1.1.0"
```

Comment:
Shouldn't use webpack for runtime, webpacking is responsibility of parent projects, not npm-distributed libraries.